README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
+
+Install/uninstall
+=================
+
+Run the command to install `sourcerer` tool on your machine:
+
+```
+curl -s https://sourcerer.io/install | sudo bash
+```
+
+Run the command to remove it from your machine:
+
+```
+curl -s https://sourcerer.io/uninstall | sudo bash
```

Comment:
This worries me. Sourcerer will inevitably change, so will the uninstall script. And the version matching will become tricky. I think you should be installing the uninstall script locally during the installation.